### ISSUE TITLE
Message Board

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-intranet-theme#12]
I have a [message board](https://intranet.1.jccmi.edu/discussion/) page. I talked to Steve and he said he would do some research on a simple message board. Basically we just need a simple message board so that employees can ask questions and have discussions.

I would also like for the latest topics to populate on the [dashboard](https://intranet.1.jccmi.edu/home/) page like the upcoming events.

**Comment by d4mation** [comment]
I've installed BBPress on the Intranet site which will allow you to create as many Message Boards/Discussion Boards as needed.

You can find all the Shortcodes available [here](https://codex.bbpress.org/features/shortcodes/), which should allow you to add a list of Topics to the Home Page and to include your Message Boards wherever necessary (Things like maybe per-Department, etc).

There doesn't appear to be much configuration that would need to be done, but here is the Settings page in case you'd want to tweak anything: https://intranet.1.jccmi.edu/wp-admin/options-general.php?page=bbpress

Here's where Forums themselves can be managed:

![image](https://user-images.githubusercontent.com/7770631/56296588-cf73ab00-60fc-11e9-81c2-f9860db5253e.png)

Basically, you create a Forum which holds Discussion Posts (Topics) and then people can Reply to them. 

I've created a Test Forum and made a Topic and a Reply as an example. Feel free to delete and replace it: https://intranet.1.jccmi.edu/discussion/

**Comment by banksashleym** [comment]
This is perfect! This is probably a loaded question, but is there any way to pull in an empoyee's picture that we have in the directory for their avatar image?

**Comment by d4mation** [comment]
It does now :)

**Comment by d4mation** [comment]
It relies upon their login email address matching what is stored in the Staff entry on jccmi.edu, but I figure it should be in most cases.

**Comment by banksashleym** [comment]
You are seriously the best!! This is so cool :)
